HW1/P4/P4.txt

```diff
@@ -1 +1,29 @@
-# Your discussion here
+CAPTAIN AMERICA has 6408 connected nodes.
+18 of the characters are not connected. 
+
+MISS THING/MARY has 7 connected nodes.
+6419 of the characters are not connected. 
+
+ORWELL has 9 connected nodes.
+6417 of the characters are not connected. 
+
+I represented my graph by first defining an RDD of (K,V) of graph edges:
+A->B
+A->C
+A->D
+B->C
+B->F
+...
+I then created an RDD of (K,V) with all the unique characters, and their associated level in the graph (graph rank). If they were not assigned a level, V=None. Initially, one character/node is designated the start node and assigned V=0. 
+
```

Comment:
Nice use of edge RDDs. I was very focused on using adjacency graphs. Good to see different approaches.